install.sh

```diff
@@ -5,14 +5,30 @@ clear
 figlet install
 
 
-# Build custom giconfig file
-echo What is your name?
-read committer_user
-echo What is your email?
-read committer_email
-sed "s/{{committer_user}}/$committer_user/g" <./gitconfig.dist >./temp
-sed "s/{{committer_email}}/$committer_email/g" <./temp >./gitconfig
-
+if [ $# -eq 0 ]
+  then
+    echo "Install starts ..."
+  else
+    echo $1
+    if [ $1 == "--gitconfig" ]
```

Comment:
if .gitconfig not exist you can always run `. ./update-gitconfig.sh`

update with this

`if [ $1 == "--gitconfig" ] || [ ! -f ~/.gitconfig ]`

Reply:
Could you please open a PR for this?